contracts/OrganizationInterface.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity ^0.4.23;
```

Comment:
An idea: Organized does not need to know about all of these functions. It would be more bytecode economical, I think, to split the interface up into the functions that are onlyOrganization and those that aren't. Organization inherits both interfaces and Organized inherits the interface that is just isWorkers.

Reply:
or we can have only 1 interface OrganizationIsWorkerInterface which Organized will import. TR, UBT, VBT can integrate with OrganizationIsWorkerInterface and work. OrganizationInterface was created to integrate with TR, UBT, VBT. However we don't need it at this  point. What do you think?

Reply:
I think I need to see that diagrammed out. Please add your proposal to the spec document, @abhayks1.